src/main/java/nextstep/ladder/model/Participants.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.ladder.model;
+
+import nextstep.ladder.model.value.Participant;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Participants {
+
+    private static List<Participant> participants;
+
+    private Participants(List<Participant> inputData) {
+
+        this.participants = inputData;
+    }
+
+    public static Participants from(String inputData) {
+        return new Participants(
+                Arrays.stream(inputData.split(","))
+                        .map(String::trim)
+                        .map(Participant::new)
+                        .collect(Collectors.toList()));
+    }
+
+    public int size() {
+        return participants.size();
+    }
+
+    public List<Participant> getParticipants() {
+        return Collections.unmodifiableList(participants);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Participants that = (Participants) o;
+        return Objects.equals(participants, that.participants);
+    }
```

Comment:
equals를 재정의 해주셨는데요!
재정의해주신 로직을 확인해보았는데 동일한 객체' 비교를 하기때문에 true로만 반환 됩니다. 
아래 처럼 정의해주시면 어떠실지 의견드립니다.
```suggestion
   @Override
    public boolean equals(Object obj) {
        return super.equals(obj);
    }
```

Reply:
'동일한 객체' 비교를 하다는 말이 현재 사용하고 있는 로직에서 말씀하시는건지요..? equals와 hashcode같은 경우 정확히 언제 사용을 해야할지에 대한 정의를 내리지 못하겠네요..